db/migrate/20160321160404_add_middleware_event_streams.rb

```diff
@@ -0,0 +1,7 @@
+class AddMiddlewareEventStreams < ActiveRecord::Migration[5.0]
+  def change
+    add_column :event_streams, :middleware_type, :string   # relevant type (entity class name)
+    add_column :event_streams, :middleware_id, :bigint     # MIQ entity id
```

Comment:
Oh, so you're looking for a connection to a middleware_server...In that case, the column should be named middleware_server_id and middleware_server_type.  When I saw middleware but itself, I assume you were connecting to the EMS (which is already defined as we've said).